flow/project.py

```diff
@@ -2896,7 +2775,11 @@ def _incomplete(status_entry):
             # get parameters info
 
             def _add_parameters(status):
-                statepoint = self.open_job(id=status["job_id"]).statepoint()
+                aggregate_id = status["aggregate_id"]
+                if aggregate_id.startswith("agg-"):
+                    # TODO: Fill parameters with empty values (or shared values?)
```

Comment:
Is this TODO regarding the error message, or deciding how this actually should behave for aggregates once we implement it?

Reply:
This can be deferred, I just didn't know what to do if the user requests to show state point parameters and aggregates exist. It might make sense to show common parameters if we put aggregates with the rest of the rows of jobs...? Regardless, that's a hairy design question and can be left as TODO for the purposes of this PR.

Reply:
I'll make an issue from this as well, then.